libfreerdp/core/settings.c

```diff
@@ -926,14 +926,8 @@ BOOL freerdp_settings_copy(rdpSettings* _settings, const rdpSettings* settings)
 	CopyMemory(_settings->MonitorIds, settings->MonitorIds, 16 * sizeof(UINT32));
 	_settings->ReceivedCapabilities = malloc(32);
 
-	if (!_settings->ReceivedCapabilities)
```

Comment:
sorry, malloc does not guarantee a non NULL return

Reply:
restored